va_explorer/va_data_management/management/commands/dhis_rms.py

```diff
@@ -0,0 +1,498 @@
+"""
```

Comment:
Looking at the code structure I think this might have been a local copy fro openva_pipeline for testing? I would recommend removing in favor of only keeping `run_dhis.py` which correctly calls openva_pipeline as a pip package from the requirements.

Reply:
done